Contest_API.md

```diff
@@ -1463,17 +1463,17 @@ Properties of state objects:
 These state changes must occur in the order listed in the table above,
 as far as they do occur, except that `thawed` and `finalized` may occur
 in any order. For example, the contest may never be frozen and hence not
-thawed either, or, it may be finalized before it is thawed. That, is the
+thawed either, or, it may be finalized before it is thawed. I.e., the
```

Comment:
Nitpick: I think it would be better to write `That is` than `I.e.` since not everyone (especially non-native readers) clearly know the difference between `i.e.` and `e.g.`. But I can also do a separate PR globally rewriting these.

Reply:
Not sure that I agree that it should be done, but I agree that if it should be done it should be a separate PR fixing it globally (and it should also be backported)